Source/WebKit/Configurations/Base.xcconfig

```diff
@@ -117,7 +117,8 @@ SUPPORTS_MACCATALYST = YES;
 // Can't enable objcxx mode until rdar://135979809 is resolved.
 SWIFT_OBJC_INTEROP_MODE = objc;
 SWIFT_VERSION = 5.0;
-OTHER_SWIFT_FLAGS = $(inherited) $(OTHER_SWIFT_FLAGS$(WK_MACOS_1400)) -no-warnings-as-errors -Xfrontend -experimental-spi-only-imports -Xfrontend -enable-experimental-concurrency;
+OTHER_SWIFT_FLAGS_ADDITIONS = @$(WK_WEBKITADDITIONS_HEADERS_FOLDER_PATH)/WebKit/AdditionalResources/availability-definitions.txt
+OTHER_SWIFT_FLAGS = $(inherited) $(OTHER_SWIFT_FLAGS$(WK_MACOS_1400)) -no-warnings-as-errors -Xfrontend -experimental-spi-only-imports -Xfrontend -enable-experimental-concurrency $(OTHER_SWIFT_FLAGS_ADDITIONS);
```

Comment:
nits:
- Not sure "ADDITIONS" feels right, unless the goal is for the file to eventually contain arbitrary Internal-only swift flags. I think I'd name it OTHER_SWIFT_FLAGS_AVAILABILITY.
- We usually write the top-level setting first, and then the specializations below. xcconfig is not an imperative language, so we can write declarations in any order.

Also, you'll need to use USE_INTERNAL_SDK here. So if you put all of this feedback together, it would look like this:

```suggestion
OTHER_SWIFT_FLAGS = $(inherited) $(OTHER_SWIFT_FLAGS$(WK_MACOS_1400)) -no-warnings-as-errors -Xfrontend -experimental-spi-only-imports -Xfrontend -enable-experimental-concurrency $(OTHER_SWIFT_FLAGS_AVAILABILITY_$(USE_INTERNAL_SDK));

OTHER_SWIFT_FLAGS_AVAILABILITY_YES = @$(WK_WEBKITADDITIONS_HEADERS_FOLDER_PATH)/WebKit/AdditionalResources/availability-definitions.txt

```